docs/source/upgrade-guide.md

```diff
@@ -803,7 +803,7 @@ Changelog:
 
 Pull Request:
 
-- <https://github.com/plone/plone.restapi/pull/428>
+- `https://github.com/plone/plone.restapi/pull/428`
```

Comment:
This should be a clickable link, not an inline literal. Either revert or remove the unnecessary brackets.

```suggestion
- https://github.com/plone/plone.restapi/pull/428
```